action/init.go

```diff
@@ -22,6 +22,9 @@ const (
 	UNSTAKE  Type = 0x12
 	WITHDRAW Type = 0x13
 
+	//network network_delegation
+	NETWORKDELEGATE Type = 0x04
```

Comment:
do u want to name it as `NETWORK_DELEGATE` for better readability? and maybe we can use 0x5x... for all the network delegation

Reply:
Yeah , I think ill change it to ADD_NETWORK_DELAGATION , 
the others can be WITHDRAW_NETWORK_DELEGATION and REMOVE_NETWORK_DELEGATION